src/main/java/blackjack/domain/card/ParticipantCards.java

```diff
@@ -8,7 +8,7 @@
 
 public class ParticipantCards {
```

Comment:
```suggestion
public class Cards {
```

1단계에서 놓쳤는데, 이렇게만 네이밍해도 될 것 같아요 ㅎㅎ